webpack.config.js

```diff
@@ -1,10 +1,41 @@
-module.exports = (env, argv) => {
+module.exports = async (env, argv) => {
     const HtmlWebpackPlugin = require("html-webpack-plugin");
     const FaviconsWebpackPlugin = require("favicons-webpack-plugin");
     const { CleanWebpackPlugin } = require("clean-webpack-plugin");
     const WorkboxPlugin = require("workbox-webpack-plugin");
+    const webpack = require("webpack");
+    const { execSync } = require("child_process");
+    const fetch = require("node-fetch");
+    const moment = require("moment");
     const path = require("path");
 
+    const getContributorsForRepo = async (repoName) => {
+        const contributorsData = await fetch(`https://api.github.com/repos/LiveSplit/${repoName}/contributors`);
+        const contributorsJson = await contributorsData.json();
+        return contributorsJson.filter((user) => user.type === "User");
+    }
+
+    const lsoContributors = await getContributorsForRepo("LiveSplitOne");
+    const coreContributors = await getContributorsForRepo("livesplit-core");
+    const allContributors = lsoContributors.reduce(
+        (accumulator, currentUser) => {
+            const existingUser = accumulator.find((user) => user.login === currentUser.login);
```

Comment:
That almost makes me feel like we should be using a map instead of an array here. This is O(n²), though it probably doesn't matter much.

Reply:
Also I feel like reduce is super bloaty, this would likely be shorter (and faster) as a normal loop.

Reply:
I changed my approach to use loops instead. Let me know if you still see ways to improve it.

Reply:
Seems good this way.